src/main/java/com/fontys/api/queries/UserQuery.java

```diff
@@ -0,0 +1,22 @@
+package com.fontys.api.queries;
+
+import com.coxautodev.graphql.tools.GraphQLQueryResolver;
+import com.coxautodev.graphql.tools.GraphQLResolver;
+import com.fontys.api.entities.User;
+import com.fontys.api.service.UserService;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class UserQuery implements GraphQLQueryResolver
+{
+    private final UserService userService;
+
+    public UserQuery(UserService userService) { this.userService = userService; }
+
+    public List<User> users(int count) { return this.userService.getAllUsers(); }
```

Comment:
We're not using pagination, so 'int count' is not needed

Reply:
Fixed the problem. Is it good now?